main_test.go

```diff
@@ -137,18 +138,19 @@ webhook:
 	if err := os.WriteFile(waitForPodsReadyEnabledConfig, []byte(`
 apiVersion: config.kueue.x-k8s.io/v1alpha2
 kind: Configuration
-namespace: kueue-system
-health:
-  healthProbeBindAddress: :8081
-metrics:
-  bindAddress: :8080
-leaderElection:
-  leaderElect: true
-  resourceName: c1f6bfd2.kueue.x-k8s.io
 waitForPodsReady:
   enable: true
-webhook:
-  port: 9443
+`), os.FileMode(0600)); err != nil {
```

Comment:
nit: I think testing every API field in this package is an overkill. A test in apis/config/v1alpha2 should be enough.

The test in this file is to make sure the defaulting pipeline is properly enabled in main.go, but it shouldn't get into the details of all possible defaults.

Reply:
I see, it makes sense, I deleted one test here on waitForPodsReady and covered the defaulting logic in efaults_test.go`.